app/src/main/java/com/Alkemy/alkemybankbase/presentation/SendViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.Alkemy.alkemybankbase.presentation
+
+import androidx.lifecycle.ViewModel
+import com.Alkemy.alkemybankbase.repository.send.SendRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class SendViewModel @Inject constructor(private val sendRepo : SendRepository) : ViewModel() {
+
+    fun validateForm(toAccount_id:String,concept:String,amount:Int,date:String,currency:String,password:String,confirmPassword:String){
```

Comment:
```suggestion
    fun validateForm(toAccount_id:Int,concept:String,amount:Int,date:String,currency:String,password:String,confirmPassword:String){
```
El id de un usuario es Int:
![image](https://user-images.githubusercontent.com/97415092/201218400-a6e2ec9d-a142-4343-8cdf-3b6f71c7dd2f.png)